tests/Agent/IntegrationTests/nuget.config

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!-- For testing purposes, we want to pull in just the Sitecore logging package -->
+    <add key="Sitecore-MyGet.org" value="https://sitecore.myget.org/F/sc-packages/api/v3/index.json"/>
+  </packageSources>
+  <packageSourceMapping>
+    <!-- key value for <packageSource> should match key values from <packageSources> element -->
+    <packageSource key="Sitecore-MyGet.org">
```

Comment:
Mentioned in a Slack thread, but looks like Sitecore is moving away from MyGet. https://support.sitecore.com/kb?id=kb_article_view&sysparm_article=KB1002999

Reply:
Thanks for the heads up! I added a ticket to fix it later: https://github.com/newrelic/newrelic-dotnet-agent/issues/1796

Reply:
My mistake, I didn't realize the new feed was already available. Fixed!